package.json

```diff
@@ -46,10 +46,11 @@
     "progress": "^2.0.0",
     "shelljs": "^0.7.0",
     "typedoc-default-themes": "^0.5.0",
-    "typescript": "2.4.1"
+    "typescript": "^2.5.0-dev.20170727"
```

Comment:
We need to use exact versions, but I won't be merging anything that's pre-release. You need to either build a pre-release pipeline to support this into the future, or wait for a stable release.